Cargo.toml

```
@@ -96,7 +96,7 @@ libp2p-autonat = { version = "0.10.0", path = "protocols/autonat", optional = tr
 libp2p-core = { version = "0.39.0", path = "core" }
 libp2p-dcutr = { version = "0.9.0", path = "protocols/dcutr", optional = true }
 libp2p-floodsub = { version = "0.42.0", path = "protocols/floodsub", optional = true }
-libp2p-identify = { version = "0.42.0", path = "protocols/identify", optional = true }
+libp2p-identify = { version = "0.42.1", path = "protocols/identify", optional = true }
```

```suggestion
libp2p-identify = { version = "0.42.0", path = "protocols/identify", optional = true }
```

Unless we rely on the newly introduced code, I would suggest to not bump patch versions here.

---

Sorry, I thought that for every pull request it is needed to bump the versions as per the document liked in the pull request template ([Link](https://github.com/libp2p/rust-libp2p/blob/master/docs/release.md#development-between-releases)).